pom.xml

```diff
@@ -831,6 +831,22 @@
                 <version>1.9.1</version>
```

Comment:
you need tests for that. Please setup a docker container with kafka schema registry in tests and use it.

Reply:
Absolutely! Will do.